### DISCUSSION TITLE
2023/08/11/Understanding-a-Simple-Example-of-epoll-Usage-in-Linux/

### DISCUSSION THREAD

**Discussion by giscus** [zjhzzy/zjhzzy.github.io#4] [Announcements]
# 2023/08/11/Understanding-a-Simple-Example-of-epoll-Usage-in-Linux/

这一篇博客是简单解释linux下的epoll epoll是linux提供的一种I/O事件通知机制，能够高效的处理文件描述符[1]的I/O事件，其基于事件驱动 常用函数解释int epoll_create(int size) epoll_create函数用来创建epoll的，需要传入一个int类型的变量用来指定最多监听几个描述符，如果没有出现错误便会返回一个文件描述符也就是int

https://zjhzzy.github.io/2023/08/11/Understanding-a-Simple-Example-of-epoll-Usage-in-Linux/

**Comment by zjhzzy** [comment]
哇我自己写的真好（doge
